hardhat-vultisig/contracts/Token.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import {ERC20, ERC20Burnable} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Burnable.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {IApproveAndCallReceiver} from "./interfaces/IApproveAndCallReceiver.sol";
+
+/**
+ * @title ERC20Burnable based token contract
+ */
+contract Token is ERC20Burnable, Ownable {
+    string private _name;
+    string private _ticker;
+
+    constructor() ERC20("", "") {
```

Comment:
We need Name and Ticker here. Leaving "" is troublesome for etherscan to read it.

Reply:
Also, supply must be in the inputs too

Reply:
Btw later we should fork those things to another folder, not in the #vultisig folder